README.md

```diff
@@ -1,2 +1,101 @@
-# coverter-go
-Convert AsyncAPI documents from older to newer versions.
+# AsyncAPI Converter
+
+## Overview
+
+The AsyncAPI Converter converts AsyncAPI documents from version `1.0.0`, `1.1.0` and `1.2.0` to version `2.0.0-rc1`. It supports both `json` and `yaml` formats on input and output. By default the AsyncAPI Converter coverts a document into the `json` format.
+
+## Prerequisites
+
+- `go` in version 1.11+
+
+## Installation
+
+ To install the AsyncAPI Converter package, run:
+
+`go get github.com/asyncapi/converter-go`
+
+## Usage
+
+You can use the AsyncAPI Converter in terminal or as a package.
+
+### CLI
+
+To use the AsyncAPI Converter in a terminal, build the application. Run:
+
+```bash
+git clone https://github.com/asyncapi/converter-go.git
+cd ./converter-go
+go build -o=asyncapi-converter ./cmd/api-converter/main.go
+```
+
+To convert a document use the following command:
+
+```text
+asyncapi-converter <document_path> [--toYAML] [--id=<id>]
+```
+
+where:
+
+- `document_path` is a mandatory argument that is either `URL` or `file path` to asyncapi document
+- `--toYAML` is an optional argument and allows to produce results in yaml format instead of json
+- `--id` is an optional argument and allows to specify application id
+
```

Comment:
**Examples**
See the following minimal examples of the AsyncAPI Converter usage in the terminal: